!Questie/QuestieOptionsForm.lua

```diff
@@ -0,0 +1,127 @@
+function Questie:OptionsForm_Init()
+    QO_showlevel = getglobal("QuestieOptionsForm".."AlwaysShowLevelCheck")
```

Comment:
maybe put all these `QuestieOptionsForm` into a variable in case the form gets renamed.